flink-java/src/main/java/org/apache/flink/api/java/ExecutionEnvironment.java

```diff
@@ -156,7 +156,15 @@ protected void setUserClassloader(final ClassLoader userClassloader) {
 		this.userClassloader = checkNotNull(userClassloader);
 	}
 
-	protected Configuration getConfiguration() {
+	public ClassLoader getUserCodeClassLoader() {
```

Comment:
If we don't add `@Internal` annotation, these methods are implicit `@Public`, is it in purpose?

Reply:
Yes, I think this should be `@Internal`. It's only used in the `StreamExecutionEnvironment`.